### DISCUSSION TITLE
Show changelog in web interface

### DISCUSSION THREAD

**Discussion by PacPalCent** [verdaccio/verdaccio#3608] [Ideas]
Hi there! I would like to have able to edit count of the tabs in web interface and to add enother tabs. For example change log. I didn't find easy and official way for it. So it will be great if you add the possibility to do it.
<img width="896" alt="Знімок екрана 2023-02-14 о 12 40 00" src="https://user-images.githubusercontent.com/26343146/218712125-df63b8cb-b7c9-465d-9d21-b2e436cdc4e3.png">

**Comment by juanpicado** [comment]
Well, there is something new I was experimenting, components ui, here an overview

https://ui-components.verdaccio.org

I don't intend to do big changes to the UI constantly, I recommend if you want your own UI but reusing same components and create your own UI, that might be the first alternative. I haven't write any docs yet but you could start to check https://github.com/verdaccio/verdaccio/tree/master/packages/plugins/ui-theme

For the other side, read CHANGELOG files is challenging, usually is a file inside of the tarball and verdaccio does not unpack files, so unless you figure out how to, I don't know how to do it.
